classes/02class/exercises/c02-network02/raghunadhpokkalath/README.md

```diff
@@ -0,0 +1,25 @@
+# c02-network02
+
+## Commands Execution Output
+
+- Answers for `10.0.0.0/16`:
+
+|Subnet|Network|Host Range|Usable Hosts|Broadcast|AWS Reserved|
```

Comment:
I think you got the host range and usable hosts header inverted :)

Reply:
Hi @christina-roperto thanks i have corrected it